### PR TITLE
Update UI section of DG, update colours used in DG

### LINKED ISSUE #201: Update DG with GUI internal structure

### PR DESCRIPTION
**Pull request by ian-from-dover**
Closes #201

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 <div markdown="span" class="alert alert-primary">
 
-:bulb: **Tip:** The `.puml` files used to create diagrams in this document can be found in the [diagrams](https://github.com/se-edu/addressbook-level3/tree/master/docs/diagrams/) folder. Refer to the [_PlantUML Tutorial_ at se-edu/guides](https://se-education.org/guides/tutorials/plantUml.html) to learn how to create and edit diagrams.
+:bulb: <b>Tip:</b> The `.puml` files used to create diagrams in this document can be found in the [diagrams](https://github.com/se-edu/addressbook-level3/tree/master/docs/diagrams/) folder. Refer to the [_PlantUML Tutorial_ at se-edu/guides](https://se-education.org/guides/tutorials/plantUml.html) to learn how to create and edit diagrams.
```
Comment (zhongfu) [inline docs/DeveloperGuide.md:40]: wait what's the point of this lol

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -83,20 +83,20 @@ The sections below give more details of each component.
...
+The `UI` component uses the JavaFx UI framework. The layout of these UI parts are defined in matching `.fxml` files that are in the `src/main/resources/view` folder. For example, the layout of the [`MainWindow`](https://github.com/AY2122S2-CS2103T-T11-4/tp/blob/master/src/main/java/peoplesoft/ui/MainWindow.java) is specified in [`MainWindow.fxml`](https://github.com/AY2122S2-CS2103T-T11-4/tp/blob/master/src/main/resources/view/MainWindow.fxml)
 
 The `UI` component,
 
 * executes user commands using the `Logic` component.
 * listens for changes to `Model` data so that the UI can be updated with the modified data.
 * keeps a reference to the `Logic` component, because the `UI` relies on the `Logic` to execute commands.
-* depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
+* depends on some classes in the `Model` component, as it displays the `Person` and `job` objects residing in the `Model`.
```
Comment (zhongfu) [inline docs/DeveloperGuide.md:99]:
```suggestion
* depends on some classes in the `Model` component, as it displays the `Person` and `Job` objects residing in the `Model`.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ The Sequence Diagram below illustrates the interactions within the `Logic` compo
 
 ![Interactions Inside the Logic Component for the `delete 1` Command](images/DeleteSequenceDiagram.png)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+<div markdown="span" class="alert alert-info">:information_source: <b>**Note:</b> The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```
Comment (zhongfu) [inline docs/DeveloperGuide.md:119]: i think you might have messed this one up

File: src/main/java/peoplesoft/logic/commands/person/PersonDeleteCommand.java [hunk 4]
```diff
@@ -42,11 +42,11 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
-        model.deletePerson(personToDelete);
+        Person p = lastShownList.get(targetIndex.getZeroBased()); // p is the person to delete
```
Comment (zhongfu) [inline src/main/java/peoplesoft/logic/commands/person/PersonDeleteCommand.java:45]: I don't really get the point of this change, but nbd

**Review by zhongfu — APPROVED** [review]
light green on white not very legible in `docs/images/ArchitectureSequenceDiagram.png`

whats `HiddenOutside` in `docs/images/ModelClassDiagram.png` etc?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ian-from-dover) [reply docs/DeveloperGuide.md:40]: it wasnt becoming bold on the website. i saw the 2 stars on either side of the text on the website

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (ian-from-dover) [reply docs/DeveloperGuide.md:119]: good catch

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ian-from-dover) [reply docs/DeveloperGuide.md:99]: fixd

File: src/main/java/peoplesoft/logic/commands/person/PersonDeleteCommand.java (hunk 4 above)
Comment (ian-from-dover) [reply src/main/java/peoplesoft/logic/commands/person/PersonDeleteCommand.java:45]: to reflect what is depicted in the sequence diagram for persondelete 1

**Comment by ian-from-dover** [comment]
the HiddenOutsides were originally in each diagram from Addressbook, used to show arrows coming from nowhere, or outside the class. I have found a solution and will be pushing it before merging

**Pull request merged**